src/Integration/NewConnectedMode/BindingConfiguration.cs

```diff
@@ -45,5 +45,32 @@ public static BindingConfiguration CreateBoundConfiguration(BoundSonarQubeProjec
         public BoundSonarQubeProject Project { get; }
 
         public SonarLintMode Mode { get; }
+
+        public override bool Equals(object obj)
```

Comment:
I've overloaded Equals and GetHashCode so configs match if they have the same Uri, ProjectKey, OrganizationKey and Mode.

Reply:
Isn't it better to also implements `IEquatable<T>` and so the `Equals(T)` method too?